components/Navbar.vue

```diff
@@ -127,15 +131,20 @@
             </div>
             <AccountBalance v-else class="is-size-7" />
 
-            <hr aria-role="menuitem" class="dropdown-divider mx-4" />
+            <hr aria-role="menuitem" class="dropdown-divider" />
 
             <div
               aria-role="menuitem"
-              class="is-flex is-justify-content-center"
+              class="is-flex is-justify-content-space-between"
               custom
               paddingless>
+              <ConnectWalletButton
+                label="general.change_account"
+                variant="connect-dropdown"
+                class="menu-item is-size-7 mr-3 w-100"
+                @closeBurgerMenu="closeBurgerMenu" />
               <NeoButton
-                class="button is-size-7 is-capitalized"
+                class="button is-size-7 is-capitalized w-100"
```

Comment:
how does it look on desktop then ? As I uderstand `w-100` gives `max-width: 100px` to the component

Reply:
it just gives 100%, no max width